File: unical-support-interface/src/app/model/email-to-answer-dto.ts
import {EmailDto} from './email-dto';
import {ResponderResultDto} from './responder-result-dto';



export class EmailToAnswerDto extends EmailDto{
  responderResult: ResponderResultDto

  constructor(data: any) {
    super(data);
    this.responderResult = data.responderResult;
  }
}

File: unical-support-2/src/main/java/unical_support/unicalsupport2/data/dto/email/EmailToAnswerDto.java
package unical_support.unicalsupport2.data.dto.email;

import lombok.Data;
import lombok.EqualsAndHashCode;

import java.util.List;

@Data
@EqualsAndHashCode(callSuper = true)
public class EmailToAnswerDto extends EmailDto {
    List<SingleAnswerDto> singleAnswers;
}
